README.md

```diff
@@ -442,10 +444,322 @@ If you want to add test data (collections and datasets) to the Dataverse instanc
 
 $ ./add-env-data.sh
 ```
+
 Note: The above command uses the [dataverse-sample-data][dv_repo_dvsampledata_url] repository whose scripts occasionally fail, so some of the test data may not be added.
 
 <br>
 
+## Writing test cases
+
+Testing is a crucial part of the SPA development. Our React application employs three main types of tests: Unit tests,
+Integration tests, and End-to-End (e2e) tests. In this section, we'll describe each type of test and how to implement them.
+
+### 1. **Unit Tests or Component tests:**
+
+Unit tests are designed to test individual React components in isolation. In our approach, we focus on testing components
+from the user's perspective, following the principles of the Testing Library. This means:
```

Comment:
```suggestion
from the user's perspective, following the principles of the React Testing Library. This means:
```

Also, maybe move the link to React Testing Library here (from below).

Reply:
Thank you for the suggestion! I have moved the link and updated the text accordingly